css/custom.css

```diff
@@ -140,6 +142,17 @@ section header:active .slogan-real {
 	display: inline-block;
 }
 
+/*.slogan-real::after {
+    content: "!";
+    position: absolute;
+    line-height: calc(50% - 48px);
+    left: calc(55.1%);
+    margin-top: 0.5%;
+    font-size: 60px;
+    color: #fff;
+    z-index: 10;
+}*/
+
```

Comment:
This section could be removed, right? I think the bolding is nice enough. :)

Reply:
Yes. Done.